docs/parts/6-kubernetes.md

```diff
@@ -0,0 +1,105 @@
+
+![No Kubernetes Clusters](../images/no-k8s-clusters.png)
+
+### 2. Update the Cluster
+
+Click on the three lined menu in the upper left of the Prism Central UI and select `Administration` -> `LCM`. Then click the `Inventory` tab and choose `Perform Inventory`. You can leave the `Enable Auto Inventory` checkbox unselected for now. Click `Proceed`.
+
+The inventory may take a long time to complete and afterwards you will likely need to refresh the page.
+
+Then navigate to the `Updates` -> `Software` tab.
+
+We need to update the Nutanix Kubernetes Engine, you can also update other components like NCC while you are here.
+
+### 3. Deploy Nutanix Kubernetes Engine
+
+Click on the three lined menu in the upper left of the Prism Central UI and select `Services` -> `Kubernetes`. Then click `Download OS Image`. Wait until the `Download Status` is `Downloaded`.
+
+![Karbon Download Status](../images/nke-download-status.png)
+
+Now back to Clusters on the left and click `Create Kubernetes Cluster`.
+
+![Karbon Create Cluster](../images/nke-create-cluster.png)
+
+Choose `Development Cluster` as the type and click `Next`.
+
+![Karbon Development Cluster](../images/nke-development-cluster.png)
+
+Name it `k8s-demo` and click `Next`.
+
+![K8S Demo Name](../images/k8s-demo.png)
+
+Leave the Node Network as `VM Network` and the number of workers as 1 and click `Next`.
+
+![K8S Demo Resources](../images/k8s-demo-resources.png)
+
+We'll use the default networking provider, so just click `Next`.
+
+![K8s Demo Network Provider](../images/k8s-demo-network-provider.png)
+
+On the Storage Class page, leave the options as their defaults, and click `Create`.
+
+![K8s Demo Storage Class](../images/k8s-demo-storage-class.png)
+
+Click `Create`. You may need to refresh to see the cluster creation status. Now just wait for the cluster to finish deploying.
+
+### 4. Download the kubeconfig file
+
+Congrats, you now have a kubernetes cluster on your Nutanix cluster.
+Download the kuberconfig file via the browser:
+
+![Download Kubeconfig](../images/download-kubeconfig.png)
+
+Transfer it to your bastion host:
+
+```sh
+scp k8s-demo-kubectl.cfg -i $(terraform output -raw ssh_private_key) root@$(terraform output -raw bastion_public_ip):~/kubeconfig
+```
+
+### 5. Access the Kubernetes cluster
+
+Now that you have the kubeconfig file on your bastion host, you can access the Kubernetes cluster from your computer. First, SSH into the bastion host:
+
+```sh
+ssh -i $(terraform output -raw ssh_private_key) root@$(terraform output -raw bastion_public_ip)
+```
+
+Then, set the `KUBECONFIG` environment variable to point to the kubeconfig file:
+
+```sh
+export KUBECONFIG=$(PWD)/kubeconfig
+```
+
+Now you can use `kubectl` to interact with your Kubernetes cluster:
```

Comment:
I had to `snap install kubectl --classic`, but maybe I ran an older version of the TF module?